test/quantization/test_workflow_module.py

```diff
@@ -822,13 +822,89 @@ def test_histogram_observer_against_reference(self, N, bins, dtype, qscheme, red
 
 
 class TestFakeQuantize(TestCase):
-    def test_half_precision_fake_quantize(self):
+    def test_forward_half_precision_fake_quantize(self):
```

Comment:
nice.  For the tests, would it make sense to reuse the existing test code across dtypes?  Something like

```
def _test_forward_per_tensor_cachemask_impl(self, device):
  is_half_options = (True, False)
  torch_types = (torch.qint8, torch.quint8)
  test_cases = itertools.product(is_half_options, torch_types)
  for is_half, torch_type in test_cases:
    ... (test case code) ...
```

Reply:
yeah that would make sense